webapp/src/DOMJudgeBundle/Resources/views/partials/scoreboard_table.html.twig

```diff
@@ -345,35 +345,16 @@
         {% set hexColor = color | colorToHex | trim('#') %}
```

Comment:
This variable is not used anymore now, I think?

Reply:
Indeed, good catch. Can clean up even more.